### DISCUSSION TITLE
[FR] Support WebDAV

### DISCUSSION THREAD

**Discussion by NightMachinery** [TortugaPower/BookPlayer#566] [Ideas]
I have a lot of audio books on my external HDD. I have connected this HDD to my computer and I am running a webDAV server to share the drive across the local network (my WiFi LAN). I like to be able to play these books with BookPlayer. Most video players on iOS have this feature, e.g., KMPlayer. 

I can run other servers (e.g., ftp) instead of webDAV, too. BookPlayer just needs to support a single network protocol.

PS: I do have my drive listed in the Files app (via SMB sharing), but BookPlayer 1. wants to import files instead of playing directly over the network 2. Does not succeed in importing them anyway (no errors shown either).

**Comment by GianniCarlo** [comment]
Hi @NightMachinary ! yes we've talked about webDAV support in this ticket https://github.com/TortugaPower/BookPlayer/issues/362 , we're working on folder support and also make the on-disk files reflect the same structure as the digital library so there's no issues with webDAV and other services that rely on modifying/adding directly files into the disk/phone
